aostypes/aostypes.go

```diff
@@ -108,8 +108,8 @@ type ResourceInfo struct {
 }
 
 // NodeConfig node configuration.
-type NodeConfig struct {
-	NodeID    string         `json:"nodeId"`
+type NodeUnitConfig struct {
```

Comment:
`NodeConfig` sounds better.

Reply:
we already have NodeConfiguration in grpc messages